src/api/constants.js

```diff
@@ -4,3 +4,8 @@ export const TMDB_MOVIE_LISTS = {
 };
 export const TMDB_BANNER_URL =
   'https://image.tmdb.org/t/p/w1920_and_h800_multi_faces/';
+
+export const TMDB_THUMBNAIL_URL =
+  'https://media.themoviedb.org/t/p/w440_and_h660_face/';
+
```

Comment:
요건 간단한건데 해당 URL이 옛날거인가봐요. 스크린샷과 같이 Moved Permanently로 리다이렉트 시키더라구요. 
이렇게 바꾸면 바로 가요! ㅋㅋ
```suggestion
export const TMDB_THUMBNAIL_URL =
  'https://image.tmdb.org/t/p/w440_and_h660_face/';

```
<img width="1018" alt="스크린샷 2024-10-23 오전 12 13 48" src="https://github.com/user-attachments/assets/91d08dc3-3754-469e-a85d-ae32be46203d">